gss-serv.c

```diff
@@ -56,12 +56,12 @@ static ssh_gssapi_client gssapi_client =
 ssh_gssapi_mech gssapi_null_mech =
     { NULL, NULL, {0, NULL}, NULL, NULL, NULL, NULL};
 
-#ifdef KRB5
+#if defined(KRB5) || defined (GSSAPI_SSPI)
```

Comment:
possible to use KRB5 without introducing a new macro ?

Reply:
Negative -- I looked pretty hard, too, since I think was the only file we really needed to change.  If you define KRB5 then a whole bunch of other conditional code ends up getting pulled in that we don't want.